autotest/pymod/gdaltest.py

```diff
@@ -547,7 +547,7 @@ def testOpen(self, check_prj = None, check_gt = None, gt_epsilon = None, \
                 # Tested on Linux, Mac OS X and Windows, with Python 2.3/2.4/2.5.
                 sv = str(new_stat[i]).lower()
                 if sv.find('n') >= 0 or sv.find('i') >= 0 or sv.find('#') >= 0:
-                    post_reason( 'NaN or Invinite value encountered '%'.' % sv )
+                    post_reason( 'NaN or Invinite value encountered ' % '.' % sv )
```

Comment:
This change and the bottom one are incorrect. But this formatting specifier should likely be fixed to %s I guess ?

Reply:
It's still not clear what should happen. The first `%` is outside quotes. See #404

Reply:
Thanks, please also rebase